staging/src/k8s.io/kube-aggregator/pkg/controllers/status/available_controller.go

```diff
@@ -264,7 +264,8 @@ func (c *AvailableConditionController) sync(key string) error {
 
```

Comment:
should probably do `u.Path = "/apis/"+apiService.Spec.Group+"/"+apiService.Spec.Version` instead, rather than appending to whatever path was included in the URL returned  from ResolveEndpoint.

(we also ignore the returned path when proxying at https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy.go#L140)